### PR TITLE
Add dp notify message to storybook

### PR DESCRIPTION
**Pull request by ahmad-demos**
- add DpNotifyMessage to Storybook.
- remove DpNotifyContainer.
- this component has already been moved to the Core.
- remove import DpNotifyContainer.

### CODE REVIEW

**Review by spiess-demos — CHANGES_REQUESTED** [review]
I propose to change two more things with this PR:

- rename component to DpNotification
- move component and its docs into ./src/DpNotification

**Comment by ahmad-demos** [comment]
> I propose to change two more things with this PR:
> 
>     * rename component to DpNotification
> 
>     * move component and its docs into ./src/DpNotification

Done in https://github.com/demos-europe/demosplan-ui/pull/537/commits/f75bdde92c62acedade07a5de0f86d2442a35662

File: src/components/DpNotification/DpNotification.vue [hunk 1]
```diff
@@ -24,10 +24,10 @@
 </template>
 
 <script>
-import { prefixClassMixin } from '../../../mixins'
+import { prefixClassMixin } from '../../mixins'
```
Comment (spiess-demos) [inline src/components/DpNotification/DpNotification.vue:27]:
```suggestion
import { prefixClassMixin } from '~/mixins'
```

File: tests/DpNotification.spec.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
-import DpNotifyMessage from '~/components/core/notify/DpNotifyMessage'
+import DpNotification from '~/components/DpNotification/DpNotification'
```
Comment (spiess-demos) [inline tests/DpNotification.spec.js:1]:
```suggestion
import DpNotification from '~/components/DpNotification'
```

**Review by spiess-demos — CHANGES_REQUESTED** [review]
some nitpicks only

File: src/components/DpNotification/DpNotification.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+# Notification
+
+The Notification component used to display brief messages or alerts to users.
+
+These messages are typically used to provide feedback, convey information, or alert users about specific events or actions.
+
+## Custom usage
+
+```html
+<dp-notify-message
```
Comment (spiess-demos) [inline src/components/DpNotification/DpNotification.stories.mdx:18]:
```suggestion
<dp-notification
```

File: src/components/DpNotification/DpNotification.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+import DpNotification from './'
+
+<Meta
+    title="Components/Notification"
+    component={DpNotification}
+/>
+
+# Notification
+
+The Notification component used to display brief messages or alerts to users.
```
Comment (spiess-demos) [inline src/components/DpNotification/DpNotification.stories.mdx:11]:
```suggestion
The Notification component is used to display brief messages or alerts to users.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,13 +5,14 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 ## UNRELEASED
 
 ### Removed
+- ([#537](https://github.com/demos-europe/demosplan-ui/pull/537)) Remove DpNotifyContainer component ([@ahmad-demos](https://github.com/@ahmad-demos))
 - ([#536](https://github.com/demos-europe/demosplan-ui/pull/536)) Remove DpToggleForm component ([@ahmad-demos](https://github.com/@ahmad-demos))
 
 ### Changed
```
Comment (spiess-demos) [inline CHANGELOG.md:11]: I think we have to mention here that the component has been renamed, which is a breaking change.

**Review by spiess-demos — CHANGES_REQUESTED** [review]

**Review by spiess-demos — APPROVED** [review]
nice! lgtm.

**Pull request merged**